### ISSUE TITLE
how to get the Dataset

### ISSUE THREAD

**Issue by wuying484** [UMich-BipedLab/LiDAR_intrinsic_calibration#2]
I am sorry,I can't get your Dataset. I need to apply for verification. How can I get the Dataset?

**Comment by brucejk** [comment]
Hi, 

Thank you for your interest in this package! I will try my best to help you. 

The datasets can be downloaded [here](https://github.com/UMich-BipedLab/LiDAR_intrinsic_calibration#dataset).

Could you explain more what do you mean by "You can't get the datasets?"

**Comment by wuying484** [comment]
> Hi,
> 
> Thank you for your interest in this package! I will try my best to help you.
> 
> The datasets can be downloaded [here](https://github.com/UMich-BipedLab/LiDAR_intrinsic_calibration#dataset).
> 
> Could you explain more what do you mean by "You can't get the datasets?"


thank you，
This Datasets needs to apply for access rights, and I applied, but it didn't give me feedback.

**Comment by brucejk** [comment]
Hi, 

Sorry for the confusion! Here are the datasets:
[training set](https://drive.google.com/drive/folders/1zWWLG9hihDuV9nYf7GQvgstnJShX2zSQ?usp=sharing)
[validation set](https://drive.google.com/drive/folders/1o5VmbwKFHWNm-7X89y1GguL9fUHPiQi7?usp=sharing)

I will update the readme. Thank you for letting me know the links were not working anymore! 

Please let me know if you have any other problems! You are always welcome to create a new issue!

**Comment by wuying484** [comment]
thank you for your reply

> Hi,
> 
> Sorry for the confusion! Here are the datasets:
> [training set](https://drive.google.com/drive/folders/1zWWLG9hihDuV9nYf7GQvgstnJShX2zSQ?usp=sharing)
> [validation set](https://drive.google.com/drive/folders/1o5VmbwKFHWNm-7X89y1GguL9fUHPiQi7?usp=sharing)
> 
> I will update the readme. Thank you for letting me know the links were not working anymore!
> 
> Please let me know if you have any other problems! You are always welcome to create a new issue!

thank you for your reply
